packages/webpack/src/presets/esbuild.ts

```diff
@@ -27,16 +27,18 @@ export function esbuild (ctx: WebpackConfigContext) {
         fullySpecified: false
       },
       options: {
-        loader: 'ts',
-        target
+        target,
+        ...ctx.nuxt.options.webpack.loaders.esbuild,
+        loader: 'ts'
       }
     },
     {
       test: /\.m?[jt]sx$/,
       loader: 'esbuild-loader',
       options: {
-        loader: 'tsx',
-        target
+        target,
+        ...ctx.nuxt.options.webpack.loaders.esbuild,
```

Comment:
Good point. I'd very happily have an additional namespace - but happy to defer that until someone asks. It could be `esbuild` and `esbuildTsx` like `css` and `cssModules`.